packages/react-conformance/src/isConformant.ts

```diff
@@ -63,3 +69,40 @@ export function isConformant<TProps = {}>(...testInfo: Partial<IsConformantOptio
     }
   });
 }
+
+/**
+ * Run default and extra tests that don't require TypeScript information
+ * @param mergedOptions
+ */
+function runNonTypeTests(mergedOptions: IsConformantOptions) {
+  const { disabledTests = [], extraTests } = mergedOptions;
+
+  for (const test of Object.keys(defaultTests)) {
+    if (!disabledTests.includes(test)) {
+      const func = defaultTests[test as keyof DefaultTestObject];
+      if (isNonTypeTest(func)) {
+        func(mergedOptions);
+      }
+    }
+  }
+
+  if (extraTests) {
+    describe('extraTests', () => {
+      for (const test of Object.keys(extraTests)) {
+        if (!disabledTests.includes(test)) {
+          const func = extraTests[test];
+          if (isNonTypeTest(func)) {
+            func(mergedOptions);
+          }
+        }
+      }
+    });
+  }
+}
+
+/**
+ * Verifies that a test function has only one parameter. If so, this test does not require TS info.
+ */
+function isNonTypeTest<TProps = {}>(func: ConformanceTest<TProps>): func is BaseConformanceTest<TProps> {
+  return func.length === 1;
```

Comment:
this is not the most robust or explicit API to have for such a critical functionality. looks like missed opportunity to design it properly while introducing breaking change. would be great to bring this to v-build sync to discuss things for better outcome. ty